.github/workflows/post-merge.yaml

```diff
@@ -0,0 +1,57 @@
+name: lambda-deploy
+
+on:
+  push:
+    branches: [ 'master' ]
+    tags: [ 'v?[0-9]+\.[0-9]+\.[0-9]+' ]
+
+concurrency:
+  group: ${{ github.workflow }}
+
+jobs:
+  lambda-lint:
+    uses: "./.github/workflows/lint.yaml"
```

Comment:
this is not just linting since it includes tests, correct?

Reply:
yeah, I'll change the name